models/app.model.js

```diff
@@ -20,8 +20,13 @@ function getArticleById (chosenId) {
     })
 }
 
-function getAllArticlesModel () {
-    return db.query(`SELECT
+function getAllArticlesModel (topicToFilter) {
+    if (!["mitch", "cats", "paper", undefined].includes(topicToFilter)) {
```

Comment:
having an array do this is quite brittle and will struggle when deployed. Have a think about another way you could check whether or not a topic is already in the db :)